src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        storageDao.add(new User(1L,"login1", "123456", 50));
+        storageDao.add(new User(2L, "login2", "789012", 25));
+    }
+
+    @Test
+    void register_NotOk_userIsNull() {
+        assertThrows(IllegalArgumentException.class,
+                () -> registrationService.register(null));
+    }
+
+    @Test
+    void register_NotOk_loginFieldIsNull() {
+        assertThrows(IllegalArgumentException.class,
+                () -> registrationService.register(new User(null, "", 0)));
```

Comment:
you leave ID field empty in all test Users

Reply:
The ID is automatically assigned to the user after being added to the storage (see line 11 in the StorageDaoImpl)